packages/zipkin-instrumentation-request-promise/package.json

```diff
@@ -16,16 +16,16 @@
     "@types/request": "^2.0.8"
   },
   "dependencies": {
+    "asap": "^2.0.6",
```

Comment:
:+1:

Reply:
Jose, I set the version 03c84c680240a9210cf08e2a7c94a96d874df5db
but what is the point of fixing patch version?

Reply:
Sorry I meant `^2.0.4` but it is a patch so probably this is good enough :+1: